### DISCUSSION TITLE
Updated Lammps Instructions

### DISCUSSION THREAD

**Discussion by owen-rett** [ACEsuit/mace#184] [Q&A] [answered]
Hello, 
Noticed in issue https://github.com/ACEsuit/mace/issues/181#issue-1923915319 that MACE-Lammps is now capable of using multiple GPUs. Is there an updated set of instructions for compiling lammps / using the pair potential in a multi-gpu case? 
Thank you

**Comment by wcwitt** [comment, marked as answer]
While it's true the functionality exists, we are still testing it. Hopefully those instructions will appear soon.
